tests/sanity/ignore-2.10.txt

```diff
@@ -0,0 +1,30 @@
+scripts/inventory/zabbix.py future-import-boilerplate
+scripts/inventory/zabbix.py metaclass-boilerplate
+plugins/modules/monitoring/zabbix/zabbix_action.py validate-modules:doc-choices-do-not-match-spec
```

Comment:
In tests/sanity/ignore-2.10.txt and tests/sanity/ignore-2.9.txt could you please do:
```suggestion
plugins/modules/zabbix_action.py validate-modules:doc-choices-do-not-match-spec
```